Pipfile

```
@@ -4,25 +4,26 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+azure-identity = "~=1.17"
 better-abc = ">=0.0.3"
 marshmallow-dataclass = "~=8.5"
 numpy = ">=1.18"
 oic = "~=1.7.0"
 pandas = ">=1.4"
-requests = "~=2.14"
-typeguard = ">=2.13"
 pbr = "~=6.0"
+pyarrow = "~=17.0.0"
```

```suggestion
pyarrow = "~=17.0"
```

Since I think we only want to lock the "major part" of the version right?

> `~=17.0.0` is equivalent to `>=17.0.0, ==17.0.*`
> `~=17.0` is equivalent to `>=17.0, ==17.*`